content/docs/command-reference/remote/modify.md

```diff
@@ -250,27 +250,71 @@ For more information on configuring Azure Storage connection strings, visit
 
 Please check out
 [Setup a Google Drive DVC Remote](/doc/user-guide/setup-google-drive-remote) for
-a full guide on configuring Google Drives for use as DVC remote storage,
-including obtaining the necessary credentials, and how to form `gdrive://` URLs.
+a full guide on using Google Drive as DVC remote storage, including how to form
+`gdrive://` URLs, and high-performance/enterprise setups.
 
 - `url` - remote location URL.
 
   ```dvc
   $ dvc remote modify myremote url gdrive://root/path/to/folder
   ```
 
-- `gdrive_client_id` - Google Project's OAuth 2.0 **client ID**.
+- `gdrive_client_id` - Google Project's **client ID** for authentication with
+  OAuth 2.0.
 
   ```dvc
   $ dvc remote modify myremote gdrive_client_id <client ID>
   ```
 
-- `gdrive_client_secret` - Google Project's OAuth 2.0 **client secret**.
+- `gdrive_client_secret` - Google Project's **client secret** for authentication
+  with OAuth 2.0.
 
   ```dvc
   $ dvc remote modify myremote gdrive_client_secret <client secret>
   ```
 
+- `gdrive_use_service_account` - instructs DVC to authenticate using a
+  [service account](/doc/user-guide/setup-google-drive-remote#access-google-drive-remote-with-a-service-account)
+  instead of OAuth. Make sure that the service account has read/write access (as
+  needed) to the file structure in the remote `url`.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_use_service_account true
+  ```
+
+- `gdrive_service_account_email` - email address of the Google Project's service
+  account.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_email <service acct email>
+  ```
+
+- `gdrive_service_account_p12_file_path` - Google Project's service account
+  `.p12` file path.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_p12_file_path path/to/file.p12
+  ```
+
+- `gdrive_service_account_user_email` - email of a user account to
+  [impersonate](https://developers.google.com/admin-sdk/directory/v1/guides/delegation)
+  with the service account.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_user_email <user account email>
+  ```
+
+- `gdrive_trash_only` - configures DVC to move remote files to
```

Comment:
on GC

Reply:
This param only has an effect when using a custom CG project? I thought it was more related to shared drive/folder access permissions.